internal/models/source_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type Source struct {
+	// Required auto-generated PK.
+	ID uint64 `db:"id" json:"id"`
+
+	// Source ID in sources
+	SourceID string `db:"source_id" json:"source_id"`
+
+	// User-facing name. Required.
+	Name string `db:"name" json:"name"`
```

Comment:
Is this it: https://github.com/RedHatInsights/sources-api-go/blob/main/model/source.go#L30

If so, why this has UNIQUE constraint? We need to be careful with uniqueness because that can usually by only implemented per account, not globally.